### PR TITLE
update for 4.12 and MULTIARCH-2635

### PR DESCRIPTION
**Pull request by chandra0007**
Addresses [MULTIARCH-2635](https://issues.redhat.com//browse/MULTIARCH-2635) Make OLM Label for Descheduler Policy for v4.12.0
Adds S390, ppc support to operator hub.

Per Review

- Removed arm64 label per discussion with ARM QE
- Changed olm.skipRange to use 4.9
- Changed minKubeVersion to 1.25
- Change art.yaml per comments (4.7.0 to 4.9.0)

Signed-off-by: Chandrasekhar Bade <cbade@redhat.com>

### CODE REVIEW

**Comment by prb112** [comment]
Reached out to @manojnkumar @jaypoulz in Slack to review on behalf of multiarch

**Comment by manojnkumar** [comment]
/LGTM for master
Needs to be back-ported to the release-4.12 branch as well?

File: manifests/4.12/cluster-kube-descheduler-operator.v4.12.0.clusterserviceversion.yaml
Comment (ingvagabund) [inline manifests/4.12/cluster-kube-descheduler-operator.v4.12.0.clusterserviceversion.yaml:33]: 4.8.0 -> 4.9.0

File: manifests/4.12/cluster-kube-descheduler-operator.v4.12.0.clusterserviceversion.yaml
Comment (ingvagabund) [inline manifests/4.12/cluster-kube-descheduler-operator.v4.12.0.clusterserviceversion.yaml:95]: 1.24.0 -> 1.25.0

**Review by ingvagabund — CHANGES_REQUESTED** [review]

**Comment by ingvagabund** [comment]
If you could also please update https://github.com/openshift/cluster-kube-descheduler-operator/blob/master/manifests/art.yaml#L10-L11 to:
```
    - search: 'olm.skipRange: ">=4.9.0 <{MAJOR}.{MINOR}.0"'
      replace: 'olm.skipRange: ">=4.9.0 <{FULL_VER}"'
```

**Comment by jaypoulz** [comment]
/hold

Adding a hold flag to ensure we can control when it gets merged post updates and approval.

File: manifests/4.12/cluster-kube-descheduler-operator.v4.12.0.clusterserviceversion.yaml
Comment (chandra0007) [reply manifests/4.12/cluster-kube-descheduler-operator.v4.12.0.clusterserviceversion.yaml]: Updated per review

**Comment by ingvagabund** [comment]
@chandra0007 would you please squash the commits before the final approval?

**Pull request merged**
